data/json/items/tool/tailoring.json

```diff
@@ -160,9 +160,7 @@
     "material": [ "plastic", "steel" ],
     "symbol": ",",
     "color": "red",
-    "ammo": [ "thread" ],
-    "initial_charges": 50,
-    "max_charges": 200,
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "thread": 200 } } ],
```

Comment:
Will `tailors_kit` require a similar change?

```
"pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "thread": 400 } } ],
```

Reply:
yes and likely i have missed more json items. i'll fix this one here at least.